apps/taxes/src/modules/tax-codes/tax-code-schema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+const taxCodeSchema = z.object({
+  description: z.string(),
+  code: z.string(),
```

Comment:
I'd mark `min()` and disallow empty strings. If something is empty, should be null/undefined. Empty strings cause strange errors (`typeof description === 'string'` to check if its valid etc)

Reply:
This schema is not needed, I removed it and created a regular type instead